avro/src/test/java/com/fasterxml/jackson/dataformat/avro/EnumTest.java

```diff
@@ -127,4 +150,34 @@ public void test_avroSchemaWithString_fromStringValueToEnumValue() throws Except
         assertEquals(Gender.F, output.gender);
     }
 
+    // [dataformats-binary#388]: Default value for enums with class
+    public void testClassEnumWithDefault() throws Exception
```

Comment:
@cowtowncoder 
Test is missing `@Test` annotation.

Reply:
Good catch, will add!